src/ontology/mondo-ingest.Makefile

```diff
@@ -224,13 +224,23 @@ $(REPORTDIR)/%_excluded_terms_in_mondo_xrefs.tsv $(REPORTDIR)/%_excluded_terms_i
 	--component-signature-path $(REPORTDIR)/component_signature-$*.tsv \
 	--outpath $@
 
+# Exclusions: specific ontologies
+.PHONY: omim-intensional-exclusions-add-non-diseases
+omim-intensional-exclusions-add-non-diseases:
```

Comment:
@matentzn `make` goal for updating OMIM intensional exclusions file with non-diseases.
The file `config/omim_exclusions.tsv` already exists; we just need to update it, hence why this is a phony goal. 

I was going to make a 2nd goal for `config/omim_non_disease_exclusions.tsv`, but I was worried about it not getting updated if it already existed, so I added the download  as an additional step in the 1st / only goal.